docs/faqs/index.md

```diff
@@ -1 +1,70 @@
 # FAQs
+
+- **Is there a way to disable M3DB embedded `etcd` and just use an external `etcd` cluster?**
+Yes, you can definitely do that. It's all just about setting the etcd endpoints in config as etcd hosts instead of M3DB hosts. 
+
+- **Is there a client that lets me send metrics to m3coordinator without going through Prometheus?**
+Yes, you can use the [Prometheus remote write client](https://github.com/m3db/prometheus_remote_client_golang/).
+
+- **Why does my dbnode keep OOM’ing?**
+Refer to the [troubleshooting guide](../troubleshooting/index.md).
+
+- **Do you support PromQL?**
+Yes, M3Query and M3Coordinator both support PromQL.
+
+- **Does M3DB store both data and (tag) metadata on the same node?**
+Yes it stores the data as well as the tags since it has an embedded index. Make sure you have `IndexEnabled` set to `true` in your namespace configuration
+
+- **How are writes handled and how is the data kept consistent within M3DB?**
+M3 uses quorum/majority consistency to ensure data is written to replicas in a way that can be read back consistently. 
+For example, if you have a replication factor of 3 and your set your write and read consistencies to quorum, then all writes will only succeed if they make it to at least 2 of the 3 replicas, and reads will only succeed if they get results back from at least 2 of the 3 replicas
+
+- **Do I need to restart M3DB if I add a namespace?**
+If you’re adding namespaces, the m3dbnode process will pickup the new namespace without a restart.
+
+- **Do I need to restart M3DB if I change or delete a namespace?**
+If you’re removing or modifying an existing namespace, you’ll need to restart the m3dbnode process. It is recommended to restart one node at a time and wait for a node to be completely bootstrapped before restarting another node.
```

Comment:
Hmm, that can technically continue on and keep using the cluster right? Maybe change wording to complete the namespace deletion process?